modules/text-field/js/text-field_directive.js

```diff
@@ -93,8 +87,8 @@ angular.module('lumx.text-field', [])
                     }
 
                     $field.addClass('text-field__input');
-                    $field.on('focus', focusUpdate);
-                    $field.on('blur', blurUpdate);
+                    $field.on('focus', true, focusUpdate);
```

Comment:
is .on() second parameter is stating that the event should bubble up or not? i cant seems to find a documentation on this.